Source/Editor/Options/InterfaceOptions.cs

```diff
@@ -148,6 +148,13 @@ public enum PlayAction
         [EditorDisplay("Interface"), EditorOrder(280), Tooltip("Editor content window orientation.")]
         public FlaxEngine.GUI.Orientation ContentWindowOrientation { get; set; } = FlaxEngine.GUI.Orientation.Horizontal;
 
+        /// <summary>
+        /// Gets or sets the editor content window orientation.
+        /// </summary>
+        [DefaultValue(false)]
+        [EditorDisplay("Interface"), EditorOrder(300), Tooltip("If checked, color pickers will always modify the color unless 'Cancel' if pressed, otherwise color won't change unless 'Ok' is pressed (default)")]
+        public bool ColorPickerAlwaysChangesColor { get; set; } = false;
```

Comment:
What about like `AutoAcceptOnColorChange` or `AutoPickOnColkrChange` or `AutoAcceptColorPickerColor` or something? Idk naming things is hard.

Reply:
Well, there is nothing really automatic about the option, is it? You still pick the color, is just that the change doesn't need "acknowledgement", but not sure.

Maybe we can say "Color picker magical option 42, surprise!" or something... 😁

Reply:
That said "Auto accept color picker change" might just work. What do you think?